src/backends/fig_shapes.f90

```diff
@@ -5,6 +5,7 @@ module fig_shapes
     type, abstract :: shape
         type(RGB) :: fill_color
```

Comment:
I think it's a good idea to provide default initializers not only for the `stroke_width=1`, but also for both colors

Reply:
Already done in #33.